frontend/packages/cli/vite-plugins/set-env.ts

```diff
@@ -56,12 +66,18 @@ export function setEnvPlugin(): Plugin {
 
       const packageJsonVersion = env.npm_package_version
       const gitHash = fetchGitHash()
+      const gitBranch = fetchGitBranch()
+
+      // The main branch is considered production, all other branches are treated as previews.
+      // This alignment is done to match the deployment settings to Vercel specified in .github/workflows.
+      const envName = gitBranch === 'main' ? 'production' : 'preview'
```

Comment:
I'm not sure if I should write about it here, since it's information only our team uses.
But once it's done, it looks good!

Reply:
Indeed, there seems to be some merit to that opinion! :)   I believe managing the `envName` value within the CLI package is necessary for our team's dogfooding. Conversely, it might be a good idea to move this `//` comment to the `.github/workflows` side.